src/scvi/nn/_base_components.py

```diff
@@ -45,13 +47,26 @@ class FCLayers(nn.Module):
         Whether to inject covariates in each layer, or just the first (default).
     activation_fn
         Which activation function to use
+    encode_covariates
```

Comment:
This is not used anywhere?

Reply:
`encode_covariates` and `batch_representation` are used in Module (e.g. vae) for handling of covariates not here